eos-tech-support/eos-convert-system-qa

```diff
@@ -154,8 +154,18 @@ if $OSTREE || $APPS; then
             --set=branches="${new_branch};"
 
         # Check for an update every time the updater runs
-        # rather than once every two weeks
-	sed -i 's/IntervalDays=14/IntervalDays=0/' /etc/eos-updater.conf
+        # rather than once every two weeks. eos-updater.conf is the old
+        # location of the config file.
+        if [ -f /etc/eos-updater.conf ]; then
```

Comment:
I don't understand the purpose of checking the old file location.  If the user has the new version of the script, it means that the user has already received an os upgrade, and thus will have the config file in the new location.  In fact, I think it might be possible (though I'm not sure) that the user will have an orphaned copy of the file in the old location, that will be completely ignored, in which case it would not be helpful to edit the old file rather than the new one.